source/admin_manual/list_of_events.rst

```diff
@@ -1067,6 +1067,31 @@ A mail index file was recreated.
 .. _event_mail_index_recreated:
 
 
+indexer_worker_indexing_finished
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.15
+
+Indexer worker process completed an indexing transaction.
+
++---------------------+--------------------------------------------------------+
+| Field               | Description                                            |
++=====================+========================================================+
+| message_count       | Number of messages indexed                             |
++---------------------+--------------------------------------------------------+
+| first_uid           | UID of the first indexed message                       |
++---------------------+--------------------------------------------------------+
+| last_uid            | UID of the last indexed message                        |
++---------------------+--------------------------------------------------------+
+| user_cpu_usecs      | Total user mode cpu time of the process in             |
+|                     | microseconds at the time of sending event.             |
```

Comment:
Total user CPU spent on the indexing transaction.

(It looks like "user CPU" is more common than "user mode CPU")

Reply:
Reworded.